src/pages/Ecoverse.tsx

```diff
@@ -197,7 +203,20 @@ const EcoversePage: FC<EcoversePageProps> = ({ paths, ecoverse }): React.ReactEl
           />
         }
       >
-        <SectionHeader text={name} />
+        <SectionHeader
+          text={name}
+          editComponent={
+            permissions.edit && (
+              <SettingsButton
+                color={'primary'}
```

Comment:
The color here is `primary` but on the other pages, it's the default.
Also probably a bigger icon would look better.

Reply:
The Ecoverse page does not have banner and because of that it requires different color.

Reply:
I think those colors have a meaning and are not there just for styling reasons.

Reply:
Suggest we keep it as it is now and pick this up as part of the wider UX review / update....

Reply:
@techsmyth Agree